<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComponentSearchWidget</class>
 <widget class="QWidget" name="ComponentSearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="resetButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../tagainijisho.qrc">
         <normaloff>:/images/icons/reset-search.png</normaloff>:/images/icons/reset-search.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="currentSelection"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CandidatesKanjiList" name="candidatesList" native="true"/>
   </item>
   <item>
    <widget class="QListWidget" name="complementsList">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
     <property name="selectionRectVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CandidatesKanjiList</class>
   <extends>QWidget</extends>
   <header>gui/kanjidic2/ComponentSearchWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../tagainijisho.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>currentSelection</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
